lunes_cms/cms/migrations/0005_alter_document_word_type.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.13 on 2022-06-29 10:01
+
+from django.db import migrations, models
+from ..models.document import reverse_changes
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cms", "0004_feedback"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="document",
+            name="word_type",
+            field=models.IntegerField(
+                choices=[(0, "Nomen"), (1, "Verb"), (2, "Adjektiv")],
+                default="",
+                verbose_name="word type",
+            ),
+        ),
+    ]
+
+    operations = [
+        migrations.RunPython(reverse_changes),
```

Comment:
```suggestion
        migrations.RunPython(convert_word_type),
```
I'd rename the function